ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -340,6 +342,38 @@ export default class SecurityTab extends PureComponent {
     );
   }
 
+  renderSafeChainsListValidationToggle() {
+    const { t } = this.context;
+    const { useSafeChainsListValidation, setUseSafeChainsListValidation } =
+      this.props;
+
+    return (
+      <div ref={this.settingsRefs[2]} className="settings-page__content-row">
+        <div className="settings-page__content-item">
+          <span>{t('useSafeChainsListValidation')}</span>
+          <div className="settings-page__content-description">
+            {t('useSafeChainsListValidationDescription1') + ' '}
+            <b>chainid.network</b>
+            {' ' + t('useSafeChainsListValidationDescription2')}
+          </div>
+        </div>
+        <div className="settings-page__content-item">
```

Comment:
This class can be removed :) 
You can find examples here
https://github.com/MetaMask/metamask-extension/blob/06ff71a093d921d854d7d1b3a0a6baaff5f49e3e/ui/pages/settings/security-tab/security-tab.component.js#L213